docs/DeveloperGuide.adoc

```diff
@@ -1037,6 +1061,31 @@ Use case ends.
 +
 Use case resumes at step 2.
 
+=== Use case: Add interview date
+
+*MSS*
+
+1.  User requests to list persons
+2.  AddressBook shows a list of persons
+3.  User requests to add an interview date to the person in the list
```

Comment:
Are we using `add` or `edit` command to add interview date?

Reply:
I think actual implementation details do not matter in user stories? It is only meant for specifying the capability of the system.

Reply:
I feel the same too, a high level description in use cases would be sufficient :+1: